xml/System.Runtime.InteropServices/DllImportResolver.xml

```diff
@@ -21,11 +21,11 @@
     <ReturnType>System.IntPtr</ReturnType>
   </ReturnValue>
   <Docs>
-    <param name="libraryName">To be added.</param>
-    <param name="assembly">To be added.</param>
-    <param name="searchPath">To be added.</param>
+    <param name="libraryName">The native library to resolve.</param>
+    <param name="assembly">The assembly requesting the resolution.</param>
+    <param name="searchPath">The DllImportSearchPathsAttribute on the PInvoke, if any. Otherwise, the DllImportSearchPathsAttribute on the assembly, if any. Otherwise null.</param>
```

Comment:
```suggestion
    <param name="searchPath">The <see cref="T:System.Runtime.InteropServices.DefaultDllImportSearchPathsAttribute" /> on the PInvoke, if any. Otherwise, the <see cref="T:System.Runtime.InteropServices.DefaultDllImportSearchPathsAttribute" /> on the assembly, if any. Otherwise <see langword="null" />.</param>
```